frb/figures/finder.py

```diff
@@ -84,6 +86,9 @@ def generate(image, wcs, title, flip_ra=False, flip_dec=False,
         secondary_coord (astropy.coordinates.SkyCoord, optional):
           If provided, place a mark in cyan at this coordinate
           Assume it is an offset star (i.e. calculate offsets)
+        secondary_offset (bool, optional):
+          If True, secondary_coord is an offset star
+          Otherwise, it is considered part of a pair of sources
```

Comment:
Please specify what "it is considered part of a pair of sources", i.e. change to "Otherwise, `secondary_coord` is considered part of a pair of sources"

Reply:
revised